_posts/2015-12-10-rust-hack-and-learn-joins-ots.md

```diff
@@ -0,0 +1,19 @@
+---
+published: true
+layout: post
+title: "Rust Hack and Learn joins OTS"
+tags:
+  - global
+  - rust
+authors:
+  - florian
+image: /assets/content/rust-logo-256x256-blk.png
+---
+
+Our regular [Rust](http://rust-lang.org) Hack and Learn meetup has now [officially joined](http://www.meetup.com/opentechschool-berlin/events/227320730/) the OpenTechSchool umbrella. We meet biweekly and learn the new programming language created by Mozilla. The Rust Hack and Learn is coached by Florian Gilcher, member of the global Rust community team, Johan Hoffman, founder of the group and Florian Hartwig.
+
+We make this move because we wanted to join an organisation that supports us in our goals and provides a helpful framework. OpenTechSchool runs many events with a similar concept and promotes them, which allows us to be more effective in our outreach by doing it together with other language communities.
+
+Finally, many of us come from other language communities and see a lot of value in collaborating instead of telling juvenile jokes about each other.
```

Comment:
I know this is intended to be tongue-in-cheek, but written content doesn't transport that especially well.  Maybe hyperbole would work (s/telling juvenile jokes/fighting/), maybe just leave it off and emphasize collaboration (+= "learning from each other.")